From abfe2bdc89179a3345f899c4cdf5ff6231b58d40 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Fri, 25 Dec 2020 13:49:46 +0100
Subject: [PATCH] host-pip-requirements: add reForis distutils

According to:
https://github.com/openwrt/packages/tree/master/lang/python#installing-host-side-python-packages

This allows us to specify hash for reForis distutils on PyPI
and once we specified hash, we can then set Host package in Makefile of
reForis

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 lang/python/host-pip-requirements/reforis-distutils.txt | 3 +++
 1 file changed, 3 insertion(+)
 create mode 100644 lang/python/host-pip-requirements/reforis-distutils.txt

diff --git a/lang/python/host-pip-requirements/reforis-distutils.txt b/lang/python/host-pip-requirements/reforis-distutils.txt
new file mode 100644
index 000000000..0a90888be
--- /dev/null
+++ b/lang/python/host-pip-requirements/reforis-distutils.txt
@@ -0,0 +3 @@
+reforis-distutils==0.1.0 --hash=sha256:f95432eca1bf4faced68d20d93e00fbdd0ae63801db4c701a42d0738cd0db37b
+Babel==2.9.0 --hash=sha256:da031ab54472314f210b0adcff1588ee5d1d1d0ba4dbd07b94dba82bde791e05
+pytz==2020.4 --hash=sha256:3e6b7dd2d1e0a59084bcee14a17af60c5c562cdc16d828e8eba2e683d3a7e268
-- 
2.25.1

